test/browser.js

```diff
@@ -0,0 +1,61 @@
+if (!global.console) {
```

Comment:
IE9 only passed if I had the Dev tools open without this, because console isn't an actual object unless you have them open.